Resources/Private/FusionModule/Components/Links.List.fusion

```diff
@@ -27,12 +25,9 @@ prototype(CodeQ.LinkChecker:Component.Links.List) < prototype(Neos.Fusion:Compon
 prototype(CodeQ.LinkChecker:Component.Links.List.Link) < prototype(Neos.Fusion:Component) {
     link = '{}'
 
-    markAsDoneAction = Neos.Fusion:UriBuilder {
-        action = 'markAsDone'
-    }
-    ignoreAction = Neos.Fusion:UriBuilder {
-        action = 'ignore'
-    }
+    // TODO: link to content module to edit the node
```

Comment:
@rolandschuetz Here I need some help how to do that

Reply:
See comment below.